src/content/community/releases/_index.en.md

```diff
@@ -5,16 +5,41 @@ weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
 
-## v0.14.5
+## v0.13.7
+
```

Comment:
Its empty?

Reply:
Yes, the goal is just to import the changes from `devel` and create the placeholder for 0.13.7 release notes. I suppose I could include the note for the Libreswan downtime reduction...